support/ci/rust_tests.sh

```diff
@@ -1,8 +1,5 @@
 #!/bin/bash
 
-sudo mkdir -p /hab
-sudo chmod o+w /hab
```

Comment:
Now I remember this was the code I had to add for travis to pass otherwise the dir was created by the non-root travis user and things fell down. We'll see how travis does here.

Reply:
Hmm this may "pass" though because nothing in `common` changed and thus those tests do not run.

Reply:
Looks like that test ran (and now passed) - https://travis-ci.org/habitat-sh/habitat/jobs/483675988#L1123

Reply:
Oh I bet its the FS_ROOT changes that makes this work which is a better way to go